packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -168,6 +168,15 @@ abstract class Layer extends AbstractNode with DiagnosticableTreeMixin {
 
   bool _debugMutationsLocked = false;
 
+  /// Whether or not this layer, or any child layers, can be rasterized with
+  /// [Scene.toImage] or [Scene.toImageSync].
+  ///
+  /// If `false`, calling the above methods may yield an image which is
+  /// incomplete.
+  bool supportsRasterization() {
```

Comment:
This sounds like it should be a getter.

Reply:
It does a tree walk and doesn't cache, so making it a method follows flutter guidelines

Reply:
Right, I wrote this before I saw the other implementation that does the walk...